### PR TITLE
Add badge component

### PR DESCRIPTION
**Pull request by lluisd**
Add badge component (instad of tag component)

Addresses https://github.com/GeeksCAT/anem-per-feina-frontoffice/issues/9

### CODE REVIEW

File: src/components/Badge/Badge.vue [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        badge: true,
+        [`badge--${this.variant}`]: true
+      }
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+    .badge {
```
Comment (d-asensio) [inline src/components/Badge/Badge.vue:44]: Like the button, use a capital letter for class names :) it's just a convention

**Review by d-asensio — CHANGES_REQUESTED** [review]
Great job @lluisd just a small convention fix :)

File: src/components/Badge/Badge.vue (hunk 1 above)
Comment (lluisd) [reply src/components/Badge/Badge.vue]: Done

**Review by d-asensio — APPROVED** [review]
Awesome! 💯

**Pull request merged**